keyless/pepper/service/src/main.rs

```diff
@@ -13,75 +14,30 @@ use hyper::{
-        (&Method::GET, "/v0/vuf-pub-key") => hyper::Response::builder()
-            .status(StatusCode::OK)
-            .header(ACCESS_CONTROL_ALLOW_ORIGIN, origin)
-            .header(ACCESS_CONTROL_ALLOW_CREDENTIALS, "true")
-            .header(ACCESS_CONTROL_ALLOW_METHODS, "GET, POST, OPTIONS")
-            .header(ACCESS_CONTROL_ALLOW_HEADERS, "*")
-            .header(CONTENT_TYPE, "application/json")
-            .body(Body::from(PEPPER_V0_VUF_VERIFICATION_KEY_JSON.as_str()))
-            .expect("Response should build"),
-        (&Method::POST, "/v0/fetch") => {
-            let body = req.into_body();
-            let body_bytes = hyper::body::to_bytes(body).await.unwrap_or_default();
-            let pepper_request = serde_json::from_slice::<PepperRequest>(&body_bytes);
-            info!("pepper_request={:?}", pepper_request);
-            let pepper_response = pepper_request.map(process);
-            info!("pepper_response={:?}", pepper_response);
-            let (status_code, body_json) = match pepper_response {
-                Ok(Ok(pepper_response)) => (
-                    StatusCode::OK,
-                    serde_json::to_string_pretty(&pepper_response).unwrap(),
-                ),
-                Ok(Err(BadRequest(err))) => (
-                    StatusCode::BAD_REQUEST,
-                    serde_json::to_string_pretty(&BadPepperRequestError {
-                        message: err.to_string(),
-                    })
-                    .unwrap(),
-                ),
-                Ok(Err(InternalError(_))) => (StatusCode::INTERNAL_SERVER_ERROR, String::new()),
-                Err(err) => (
-                    StatusCode::BAD_REQUEST,
-                    serde_json::to_string_pretty(&BadPepperRequestError {
-                        message: err.to_string(),
-                    })
-                    .unwrap(),
-                ),
-            };
-            hyper::Response::builder()
-                .status(status_code)
-                .header(ACCESS_CONTROL_ALLOW_ORIGIN, origin)
-                .header(ACCESS_CONTROL_ALLOW_CREDENTIALS, "true")
-                .header(ACCESS_CONTROL_ALLOW_METHODS, "GET, POST, OPTIONS")
-                .header(ACCESS_CONTROL_ALLOW_HEADERS, "*")
-                .header(CONTENT_TYPE, "application/json")
-                .body(Body::from(body_json))
-                .expect("Response should build")
+        (&Method::GET, "/about") => {
+            build_response(origin, StatusCode::OK, ABOUT_JSON.deref().clone())
         },
+        (&Method::GET, "/v0/vuf-pub-key") | (&Method::GET, "/v1/vuf-pub-key") => build_response(
+            origin,
+            StatusCode::OK,
+            PEPPER_VUF_VERIFICATION_KEY_JSON.deref().clone(),
+        ),
+        (&Method::POST, "/v0/fetch") => handle_fetch_common(origin, req, process_v0).await,
+        (&Method::POST, "/v1/fetch") => handle_fetch_common(origin, req, process_v1).await,
```

Comment:
Why are you keeping both v0 and v1 around? We can make a breaking change, no? We are not deployed on testnet...

Reply:
we want to keep both around for the time being.  especially since ppl are developing against v0.  eventually we can remove it.